server/api/controller/funding.js

```diff
@@ -0,0 +1,54 @@
+const Controller = require('./controller');
+const Config = require('../lib/config').paymentServices;
+const axios = require('axios')
+
+let instance = axios.create({
```

Comment:
can you change `axios` to `request-promise` which we already use in the server? (used in iPlanApi.js for example)

Reply:
@florpor Apparently request-promise is deprecated
![image](https://user-images.githubusercontent.com/22238092/102971750-97904e00-4502-11eb-980a-01f9536e3f23.png)

Reply:
`bla`

Reply:
ah yeah, i see it was deprecated earlier this year :/ ...
axios it is then